db/db_basic_test.cc

```diff
@@ -140,7 +140,7 @@ TEST_F(DBBasicTest, CompactedDB) {
   ASSERT_EQ("NOT_FOUND", Get("kkk"));
 
   // MultiGet
-  std::vector<std::string> values;
+  std::vector<std::string> values(6);
```

Comment:
Did you mean to change this to PinnableSlice?

Reply:
We need test coverage for both std::string and PinnableSlice version of MultiGet so I'm inclined not to change this to PinnableSlice.